### DISCUSSION TITLE
已经选择openclash，但是编译出的固件却没有。

### DISCUSSION THREAD

**Discussion by yjiawqgj** [vernesong/OpenClash#3428] [Q&A]
在编译openwrt固件时已经在luci里选择CONFIG_PACKAGE_luci-app-openclash=y，但编译后的固件却没有openclash，怎么解决？

**Comment by LQ2002** [comment]
luci-app-openwrt是选择的*还是M？

**Reply by yjiawqgj** [reply]
我换了一个别人的仓库成功了。

**Reply by abelwang123** [reply]
我也遇到这个问题了，请问换了哪个仓库？

**Reply by yjiawqgj** [reply]
> 我也遇到这个问题了，请问换了哪个仓库？

早忘了，很长时间没有编译了。随便找一个试试吧！
